Sources/TuistCore/Utils/FileHandler.swift

```diff
@@ -136,10 +145,11 @@ public final class FileHandler: FileHandling {
 
     /// Returns true if there's a folder or file at the given path.
     ///
-    /// - Parameter path: Path to check.
+    /// - Parameters:
+    ///     - path: Path to check.
     /// - Returns: True if there's a folder or file at the given path.
     public func exists(_ path: AbsolutePath) -> Bool {
-        return fileManager.fileExists(atPath: path.pathString)
+        return FileManager.default.fileExists(atPath: path.pathString)
```

Comment:
Why not using the instance variable `fileManager` instead?

Reply:
Good catch, changed.